tests/phpunit/test-class-wc-stripe-upe-payment-gateway.php

```diff
@@ -190,6 +190,7 @@ public function test_process_payment_returns_valid_response() {
 			'description' => $description,
 			'customer'    => $customer_id,
 			'metadata'    => $metadata,
+			'statement_descriptor' => null,
```

Comment:
No big deal but I believe by convention the other values should be indented further here (and below).

Reply:
Fixed in 2702632.